src/modules/apis.ts

```
@@ -5,35 +5,75 @@ export async function getNearbyPOIs(
   longitude: number,
   type: string,
 ) {
-  let response;
+  let response: any;
   try {
     response = await fetch(
       `https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=${latitude},${longitude}&radius=1000&type=${type}&key=${GOOGLEAPIKEY}`,
     );
-
     if (response.status !== 200) {
-      throw new Error('Error fetching from API: ' + response.status);
```

Check if this should throw an error?